src/CommandLine.DocumentationGenerator/DocumentationWriter.cs

```diff
@@ -46,7 +46,7 @@ public override void WriteOptionDescription(CommandOption option)
             }
             else
             {
-            _writer.WriteString(description);
+                _writer.WriteString(description);
```

Comment:
```suggestion
            _writer.WriteString(description);
```

Reply:
I believe the formatter will fail, but I'll revert this